vllm/metadata.csv

```diff
@@ -0,0 +1,44 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
+vllm.avg.generation_throughput.toks_per_s,gauge,,,,Average generation throughput in tokens/s,0,vllm,,,
+vllm.avg.prompt.throughput.toks_per_s,gauge,,,,Average prefill throughput in tokens/s,0,vllm,,,
+vllm.cache_config_info,gauge,,,,Information on cache config,0,vllm,,,
+vllm.cache_usage_perc,gauge,,percent,,GPU KV-cache usage. 1 means 100 percent usage,0,vllm,,,
+vllm.cpu_cache_usage_perc,gauge,,percent,,CPU KV-cache usage. 1 means 100 percent usage,0,vllm,,,
+vllm.e2e.request_latency.seconds.bucket,count,,second,,Histogram of end to end request latency in seconds.,0,vllm,,,
+vllm.e2e.request_latency.seconds.count,count,,second,,Histogram of end to end request latency in seconds.,0,vllm,,,
+vllm.e2e.request_latency.seconds.sum,count,,second,,Histogram of end to end request latency in seconds.,0,vllm,,,
```

Comment:
Might be worth updating the description to say what is different between these three metrics. I can tell by the metric name that one is a count and one is a sum, but that's not clear from the description. There are similar examples below in this file where the description is repeated for the aggregations.

Reply:
I think this makes sense for duration metrics but it's a bit harder to individually define what these metrics represent when it comes to those that aren't